lib/project_types/script/commands/enable.rb

```diff
@@ -12,6 +11,7 @@ class Enable < ShopifyCli::Command
       end
 
       def call(_args, _name)
+        ShopifyCli::Tasks::EnsureEnv.new.call(@ctx, required: [:api_key, :secret, :shop])
```

Comment:
You can just use `ShopifyCli::Tasks::EnsureEnv.call`. The Task Class has a class method 
```ruby
    def self.call(*args)
      task = new
      task.call(*args)
    end
```